tests/qml/Properties.qml

```diff
@@ -1,6 +1,6 @@
-import QtQuick 2.5
+import QtQuick 2.5 as QQ
```

Comment:
The original line should be preset to keep the existing testcase (see comment below).
Perhaps use some other import, not `QtQuick`, and add it to the second line?